src/sqlancer/questdb/gen/QuestDBExpressionGenerator.java

```diff
@@ -54,6 +55,15 @@ public Node<QuestDBExpression> generateConstant() {
             return QuestDBConstant.createBooleanConstant(Randomly.getBoolean());
         case FLOAT:
             return QuestDBConstant.createFloatConstant(globalState.getRandomly().getDouble());
+        case SYMBOL:
+            StringGenerationStrategy strategy = Randomly.StringGenerationStrategy.ALPHANUMERIC;
+            // Add quotes to form a valid string for QuestDB
+            String str = '\"' + strategy.getString(globalState.getRandomly()) + '\"';
```

Comment:
I think it would be more elegant to format the symbol in the constant itself (e.g., see https://github.com/sqlancer/sqlancer/blob/40a1d26adaa0cc76050aa9380d271565862e4f95/src/sqlancer/duckdb/ast/DuckDBConstant.java#L79).